erpnext/hr/doctype/expense_claim_type_detail/expense_claim_type_detail.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
Delete expense_claim_type_detail folder, as you renamed it to expense_claim_summary